### ISSUE TITLE
[BUG] Laptop classified as container

### ISSUE THREAD

**Issue by thiagoftsm** [netdata/netdata-cloud#299]
**Describe the bug**	
My laptop is classifed as `container`, but I installed netdata direct on host.

**To Reproduce**	

1. Compile netdata from scratch.
2. Connect to cloud.
3. Get local information `/api/v1/info` and verify that you do not have any node set, this is my [info](https://github.com/netdata/netdata-cloud/files/8119657/info.txt).
4. Access the host dashboard, and verify that you have `container` on top. This is not expected when all `containers` variable are set to `none` or `unknow`.

**Expected behavior**	

When netdata is installed direct on host,  I expect to see some description that matches my hardware, instead `container`.

**Screenshots**	

![Screenshot_20220222_195228](https://user-images.githubusercontent.com/49162938/155208517-32fa3c1a-9c59-4ab7-b8eb-31016d027b2f.png)

**Error logs**	
1. Open developer devtools (e.g for [chrome](https://developers.google.com/web/tools/chrome-devtools/open)	
2. Submit a screenshot or better a [HAR file](https://support.zendesk.com/hc/en-us/articles/204410413-Generating-a-HAR-file-for-troubleshooting)	

There was not any error.

**Desktop (please complete the following information):**	
 - OS:  Slackware Current
 - Browser: Firefox
 - Version: firefox-97.0.1

**Additional context**	
This war-room is already using the latest architecture.

**Comment by thiagoftsm** [comment]
I observed that @ktsaou detected a problem analogous to this, and he also made an issue https://github.com/netdata/netdata-cloud/issues/300. 

I would like to call attention that we are running different Linux distributions. 

cc: @dionysis-netdata @cpipilas

**Comment by thiagoftsm** [comment]
I compiled the latest master branch now to give additional information:

```sh
bash-5.1# /usr/libexec/netdata/plugins.d/system-info.sh 
NETDATA_CONTAINER_OS_NAME=none
NETDATA_CONTAINER_OS_ID=none
NETDATA_CONTAINER_OS_ID_LIKE=none
NETDATA_CONTAINER_OS_VERSION=none
NETDATA_CONTAINER_OS_VERSION_ID=none
NETDATA_CONTAINER_OS_DETECTION=none
NETDATA_CONTAINER_IS_OFFICIAL_IMAGE=false
NETDATA_HOST_OS_NAME=Slackware
NETDATA_HOST_OS_ID=slackware
NETDATA_HOST_OS_ID_LIKE=unknown
NETDATA_HOST_OS_VERSION=15.0
NETDATA_HOST_OS_VERSION_ID=15.0
NETDATA_HOST_OS_DETECTION=/etc/os-release
NETDATA_HOST_IS_K8S_NODE=false
NETDATA_SYSTEM_KERNEL_NAME=Linux
NETDATA_SYSTEM_KERNEL_VERSION=5.16.10
NETDATA_SYSTEM_ARCHITECTURE=x86_64
NETDATA_SYSTEM_VIRTUALIZATION=none
NETDATA_SYSTEM_VIRT_DETECTION=none
NETDATA_SYSTEM_CONTAINER=unknown
NETDATA_SYSTEM_CONTAINER_DETECTION=none
NETDATA_SYSTEM_CPU_LOGICAL_CPU_COUNT=4
NETDATA_SYSTEM_CPU_VENDOR=GenuineIntel
NETDATA_SYSTEM_CPU_MODEL=Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz
NETDATA_SYSTEM_CPU_FREQ=2700000000
NETDATA_SYSTEM_CPU_DETECTION=lscpu sysfs
NETDATA_SYSTEM_TOTAL_RAM=16690069504
NETDATA_SYSTEM_RAM_DETECTION=procfs
NETDATA_SYSTEM_TOTAL_DISK_SIZE=1120245866496
NETDATA_SYSTEM_DISK_DETECTION=sysfs
```

**Comment by dionysis-netdata** [comment]
Released on [v2.17.2](https://github.com/netdata/cloud-frontend/releases/tag/2.17.2)
